docs/src/accelerators.md

```diff
@@ -0,0 +1,100 @@
+
+## Mathematical Understanding
+
+In practice, these acceleration methods have been found to significantly accelerate convergence on several inverse problems. However, the mathematical reasoning behind this acceleration for EKI is not yet fully understood. A potential avenue for understanding how momentum affects the EKI algorithm is by using gradient flows, as in ([Kovachki and Stuart, 2021](https://jmlr.org/papers/v22/19-466.html)).
+
+## Using Accelerators
+
+An EKI struct can be created with momentum acceleration as follows:
+
+```julia
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.ParameterDistributions
+
+seed = 1
+rng = Random.MersenneTwister(1)
+
+# assume we have defined a prior distribution, a scalar N_ens, a vector of observations y, and observational noise covariance Γ
+
+initial_ensemble = EKP.construct_initial_ensemble(rng, prior, N_ens)
+
+ekiobj_accelerated = EKP.EnsembleKalmanProcess(
+            initial_ensemble,
+            y,
+            Γ,
+            Inversion();
+            accelerator = NesterovAccelerator()
+        )
+```
+The rest of the process is the same as without the accelerator.
+
+Very similarly, one can create an ETKI struct with acceleration:
+```julia
+etkiobj_accelerated = EKP.EnsembleKalmanProcess(
+            initial_ensemble,
+            y,
+            Γ,
+            TransformInversion();
+            accelerator = NesterovAccelerator()
+        )
+```
+
+Again, the rest of the process is the same as without the accelerator.
+
+## Experiments on EKI and ETKI
+
+Accelerators have been found to accelerate EKI convergence on a number of example inverse problems. In this "exponential sine" inverse problem, we look to solve for amplitude and vertical shift parameters of an underlying sine wave. The model is defined as $\exp\left(\theta_1 \sin(\phi+t) + \theta_2\right)$, with unknown parameters $\theta = [\theta_1, \theta_2]$. Observations (model output) consist of the difference between maximum and minimum, and the mean, of the evaluated model. We define $\theta_\text{true} = \left[1, 0.8\right]$.
+
+The `NesterovAccelerator` (shown in blue) has been found to produce the most consistent acceleration on this problem, as seen below. The `FirstOrderNesterovAccelerator` (shown in red) uses a momentum coefficient very similar to that of the `NesterovAccelerator`, and enjoys similar performance. The `ConstantNesterovAccelerator` (shown in green) is effective in this test case, but can be very unstable. These methods differ only in their momentum coefficient values, which are plotted on the right. Vanilla EKI is shown in black. The experiment is repeated 50 times; ribbons denote one standard error from the mean.
```

Comment:
add the brackets in `NesterovAccelerator()` or in the Constant case, the `(0.9)` or whatever value of beta.